apps/callscreen/js/conference_group_handler.js

```diff
@@ -112,6 +107,26 @@ var ConferenceGroupHandler = (function() {
     }
   }
 
+  /* Observe changes to the node that holds the group call duration, this is
+   * also used for displaying the call ended string and needs to be resized
+   * dynamically to accomodate for certain locales. */
+  function observeMutation(mutations) {
+    if (groupDurationChildNode.hasAttribute('data-l10n-id') &&
+        groupDurationChildNode.getAttribute('data-l10n-id') === 'callEnded') {
+      /* Disable the observer to prevent it from being called recursively
+       * while we modify the DOM tree, re-enable it once we're done . */
+      mutationObserver.disconnect();
+      FontSizeManager.adaptToSpace(
+        CallScreen.getScenario(), groupLabel, false, 'end');
+      mutationObserver.observe(groupDurationChildNode, mutationObserverConfig);
```

Comment:
This is referring to ```mutationObserverConfig``` which hasn't been instantiated yet, at least in order of prose. It would be better to refactor this into a function that instantiates the config and sets the ```MutationObserver```.